rust/src/util.rs

```diff
@@ -134,7 +165,22 @@ impl Ord for Variant {
 
 impl Display for Variant {
     fn fmt(&self, f: &mut Formatter<'_>) -> fmt::Result {
-        write!(f, "{}", self.name)?;
+        write!(f, "{}:{}", self.name, self.tag)?;
```

Comment:
Hm, this is actually breaking. Let me check where we use `Display` downstream (I am pretty sure it might break STON data everywhere)...

Reply:
Change reverted